central/vulnerabilityrequest/datastore/datastore.go

```diff
@@ -71,24 +70,9 @@ func New(s store.Store, indexer index.Indexer, searcher searcher.Searcher,
 
 // NewForTestOnly returns a new instance of DataStore. TO BE USED FOR TESTING PURPOSES ONLY.
 // To make this more explicit, we require passing a testing.T to this version.
-func NewForTestOnly(t *testing.T, db *pkgRocksDB.RocksDB, bleveIndex bleve.Index,
-	pendingReqCache cache.VulnReqCache, activeReqCache cache.VulnReqCache) (DataStore, error) {
-	testutils.MustBeInTest(t)
-
-	storage := rocksdb.New(db)
-	indexer := index.New(bleveIndex)
-	d := &datastoreImpl{
-		store:           storage,
-		index:           indexer,
-		searcher:        searcher.New(storage, indexer),
-		pendingReqCache: pendingReqCache,
-		activeReqCache:  activeReqCache,
-	}
-
-	if err := d.buildIndex(context.Background()); err != nil {
-		return nil, errors.Wrap(err, "failed to build index from existing store")
-	}
-	return d, nil
+func NewForTestOnly(_ *testing.T, _ *pkgRocksDB.RocksDB, _ bleve.Index,
```

Comment:
I'm going to assume this will be removed in some subsequent PR

Reply:
correct, it'll be changed in a different PR that currently relies on it